components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,846 @@
+                                <ContextMenu menuEntries={subscriptionMenu(sub)}>
+                                    <img className="w-8 h-8 p-1" src={ThreeDots} alt="Actions" />
+                                </ContextMenu>
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+        )}
+    </React.Fragment>);
+
+    return (<div>
+        {showPaymentUI ? renderTeams() : (
+            <div className="flex flex-row">
+                <div className="flex-grow ">
+                    <h3 className="self-center">All Teams</h3>
+                    <h2>Manage teams and team members.</h2>
+                </div>
+            </div>
+        )}
+    </div>);
+}
+
+function InviteMembersModal(props: {
+    sub: TeamSubscription,
+    onClose: () => void
+}) {
+
+    const [copied, setCopied] = useState<boolean>(false);
+
+    const getInviteURL = () => {
+        const link = new URL(window.location.href);
+        link.pathname = '/subscription/'
+        link.search = '?teamid=' + props.sub.id;
+        return link.href;
+    }
+
+    const copyToClipboard = (text: string) => {
+        const el = document.createElement("textarea");
+        el.value = text;
+        document.body.appendChild(el);
+        el.select();
+        try {
+            document.execCommand("copy");
+        } finally {
+            document.body.removeChild(el);
+        }
+        setCopied(true);
+        setTimeout(() => setCopied(false), 2000);
+    };
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">Invite Members</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4 space-y-2">
+            <p className="pb-2 text-gray-500 text-base">Invite members to the team using the URL below.</p>
+
+            <div className="flex flex-col space-y-2">
+                <label htmlFor="inviteUrl" className="font-medium">Invite URL</label>
+                <div className="w-full relative">
+                    <input name="inviteUrl" disabled={true} readOnly={true} type="text" value={getInviteURL()} className="rounded-md w-full truncate pr-8" />
+                    <div className="cursor-pointer" onClick={() => copyToClipboard(getInviteURL())}>
+                        <img src={copy} title="Copy Invite URL" className="absolute top-1/3 right-3" />
+                    </div>
+                </div>
+                <p className="pb-4 text-gray-500 text-sm">{copied ? "Copied to clipboard!" : "Use this URL to join this team."}</p>
+            </div>
+
+        </div>
+        <div className="flex justify-end mt-6">
+            <button className={"ml-2 secondary"} onClick={() => props.onClose()}>Close</button>
+        </div>
+    </Modal>);
+}
+
+const quantities = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100];
```

Comment:
Based on the stats we looked at (sample size 40), I think you could safely omit the -5 steps between 10 and 50. Up to you! 😇 

<img width="375" alt="Screenshot 2021-03-31 at 11 39 24" src="https://user-images.githubusercontent.com/599268/113180540-1fe46480-9251-11eb-9641-cb6c3995e537.png">

```suggestion
const quantities = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100];
```